### ISSUE TITLE
Generate plot of cross entropy delta vs sparsity level

### ISSUE THREAD

**Issue by jds8** [jds8/SAE4DLM-CE#6]
Load the data from the saved json files and use the matplotlib library to generate a plot of cross-entropy delta versus sparsity. Try to match the second cross-entropy delta plot in Figure 3. Label your axes etc. Push the code you use to generate these plots so we can reuse it.

Attached are two examples of json files that you can use to get started. You'll have to load files like these, extract the 'delta_lm_loss(mask)' field and plot the values against the sparsity level, which is defined in the config.json file in the same directory containing the ae.pt file.

For L1, i.e. resid_post_layer_1:
Trainer 0: [delta_lm_loss(mask).json](https://github.com/user-attachments/files/26009267/delta_lm_loss.mask.json)
Trainer 1: [delta_lm_loss(mask)(1).json](https://github.com/user-attachments/files/26009266/delta_lm_loss.mask.1.json)
